src/test/acceptance/features/address-lookup.feature

```diff
@@ -74,3 +74,25 @@ Feature: Select address
       | AA1122BB |
       | A        |
       | 11AA21   |
+
+  # TODO DW HTBHF-2037 include test once address lookup is enabled
+  @ignore
+  Scenario: I am shown an error response when a bad respoonse is returned from os places and I can then enter my address manually
+    Given I have entered my details up to the postcode page
+    And OS places returns an error response
+    When I enter my postcode
+    Then I am shown the select address page
+    And I am informed that there's a problem with the postcode finder
```

Comment:
Shouldn't this be `And I am informed that there's a problem with the address lookup`?

Reply:
Updated to address lookup. I think we initially went for postcode finder because that's what the error text says.